packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/EditInvoiceDetails.tsx

```diff
@@ -99,8 +99,11 @@ const EditInvoiceDetailsContent = ({
       line_items: invoice.line_items.map((lineItem) => ({
         quantity: lineItem.quantity,
         product_id: lineItem.product.id,
-        vat_rate_id: lineItem.product.vat_rate.id,
-        vat_rate_value: lineItem.product.vat_rate.value,
+        ...(lineItem.product.vat_rate.id !== null
+          ? { vat_rate_id: lineItem.product.vat_rate.id,
+              vat_rate_value: lineItem.product.vat_rate.value,
+           }
+          : { tax_rate_value: lineItem.product.vat_rate.value / 100 }),
```

Comment:
what does the '/ 100' means? If it's the conversion from to minor units, it is a wrong way to do so.
The conversion from/to minor units depends on the currency since there is a differect decimal for some currencies.
You should use formatFromMinorUnits/formatToMinorUnits functions form our util hooks.